# Журнал изменений

## Обновления пакетов
- libffi-3.4.2
- p11-kit-0.24.0
- Python3-3.9.6
## 01.07.20201
- [Исправлено #636](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/636)
- [Исправлено #659](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/659)
- [Исправлено #662](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/662)
- [Исправлено #655](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/655)
- [Исправлено #660](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/660)
## 30.06.2021
- Исправлена опечатка в названии директории с загрузочными скриптами.
- Обновлены пакеты: 
- [Выполнено #29](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/29)
- [Выполнено #427](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/427)
- [Исправлено #629](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/629)
- [Фикс фалов со входом/выходом из chroot и исправление](https://github.com/Linux4Yourself/Linux4Yourself.Book/pull/635) бага [#634](https://github.com/Linux4Yourself/Linux4Yourself.Book/issues/634)
